style/style.scss

```diff
@@ -173,20 +171,21 @@ body {
     padding: 8px;
 }
 
-.flex-container{
-    font-size: $standard-font-size;
+.flex-container {
+    font-size: 1.5rem;
     color: $standard-text-color;
-    @include flex-wrap();
-    weight: 100%
+    width: 100%;
+    @include flex-wrap ()
 }
-
-.item {
-    width: 30%;
-    height: 100%;
-    padding: 0px 60px 15px 90px;
-    text-align: center;
+.product-info {
+    width: 20%;
+    min-width: 350px;
```

Comment:
czy dobrze rozumiem, że ten min-width to żeby ładnie się wyświetlało? Nie wiem jak to się będzie miało do ekranów <320px